packages/hash/api/src/graph/system-types.ts

```diff
@@ -254,13 +216,8 @@ const responsibilityPropertyTypeInitializer = propertyTypeInitializer({
   actorId: systemAccountId,
 });
 
-const ofOrgLinkEntityTypeInitializer = entityTypeInitializer({
-  ...types.linkEntityType.ofOrg,
-  actorId: systemAccountId,
-});
-
-const hasMembershipLinkEntityTypeInitializer = entityTypeInitializer({
-  ...types.linkEntityType.hasMembership,
+const orgMembershipLinkEntityTypeInitializer = entityTypeInitializer({
+  ...types.linkEntityType.orgMembership,
   actorId: systemAccountId,
 });
```

Comment:
Isn't this missing the `responsibility` property?

Reply:
Thanks, added in 393d2609e5f40820e98a303d43a5b9d522dd37fe :)